docs/modules/ROOT/pages/advanced-how-tos/managing-multiple-versions.adoc

```diff
@@ -1,25 +1,24 @@
+* As described above, starting a new development stream involves creating a large  amount of  *Application*, *Component* and other resources. The Project Controller helps to streamline that by allowing to create a *ProjectDevelopmentStreamTemplate* resource that specifies the resources to be created and allows for using variables to customize them. Using a template, many similar development streams can be created quickly.
 
 == Using the Project Controller
 
-In the sections below we will describe how to use the custom resources supported by the Project Controller to manage multiple software versions and generate Application and Component resources.
+In the sections below we will describe how to use the custom resources supported by the Project Controller to manage multiple software versions and generate Application, Component and other resources.
 
 === Before you begin
 
-It's possible to have the Project Controller create all the Application and Component resources you would need. It is recommended, however, for the first time a particular repository is on-boarded, to do so manually via the {ProductName} UI. That would help ensuring the right access credentials are in place as well as that the Tekton Pipeline-as-Code files are committed to the repo.
+The typical process of onboarding a componnent to {ProductName} includes having the {ProductName} Build Service generate a PR to the component Git repository which includes the Tekton Pipeline-as-Code (PaC) pipeline definition for building that component. As a part of this process various other resources such as the PaC Repository resource and Secret resources for accessing the repository are also created.
 
-When creating components the Project Controller assumes that:
+Whe using the Project Controller to create components the process of generating the PR is skipped because it is assumed the branches being created in the component Git repositores already contain PaC pipeline files. Eert Git repository, however, needs to undergo this process at least once, therefore it is recommended to onboard at least once component from each repository from the UI or by other means before attempting to manage components in that repository via the Project Controller.
 
-* The default pipeline is not being used and that each branch in a component's repo contains the necessary Tekton Pipeline-as-Code files to build it.
-* Any credentials required to access the component repo had already been configured as secrets in the {ProductName} workspace.
+The rest of this document is going to assume that process is being followed and would, on occasion requst that data would be copied from the component that was already on-boarded by other means.
```

Comment:
```suggestion
The rest of this document is going to assume that process is being followed and would, on occasion request that data would be copied from the component that was already on-boarded by other means.
```

What is "that process"? What is making the request that data is copied?

Reply:
The process here refers to the paragraph above - the process of manually onboarding the first component from a repo before using the project-controller for the rest.

The requests to copy data would be done by this document, though I suppose we could come up with a more accurate language, since we do not actually ask the user to copy any data, instead we assume data is copied by creating a branch and then needs to be adjusted.